app/src/main/res/layout/card_city_item.xml

```diff
@@ -33,10 +33,11 @@
             android:layout_height="24dp"
```

Comment:
Bilo bi dobro da je klikabilno podrucje barem 48x48. To su neki googleovi guidelinesi za lakse koristenje aplikacije